common/src/main/java/com/microsoft/identity/common/internal/ui/browser/BrowserSelector.java

```diff
@@ -77,9 +79,42 @@ public static Browser select(final Context context, final List<BrowserDescriptor
         throw new ClientException(ErrorStrings.NO_AVAILABLE_BROWSER_FOUND, "No available browser installed on the device.");
     }
 
+    private static boolean matches(@NonNull final BrowserDescriptor browserDescriptor,
```

Comment:
It is strange that this moved.  Should we have a platform-dependent variant of BrowserDescriptor?  It looks like we're establishing a need for a platform-dependent BrowserSelector instead.

Reply:
I see this called out in the description.